tests/testthat/test-object_name_linter.R

```diff
@@ -2,58 +2,53 @@ context("object_name_linter")
-  expect_equivalent(matches_styles("ihave1cat"    ), c(FALSE,  TRUE,  TRUE,  TRUE,   TRUE, FALSE))
-  expect_equivalent(matches_styles("IHAVE1CAT"    ), c( TRUE, FALSE, FALSE, FALSE,  FALSE,  TRUE))
-  expect_equivalent(matches_styles("I.HAVE_ONECAT"), c(FALSE, FALSE, FALSE, FALSE,  FALSE, FALSE))
-
-  expect_error(lintr:::matches_styles("text", "invalid_style"), "Invalid style")
+  expect_equivalent(lapply(styles, check_style, nms = "IHave1Cat"    ), c( TRUE, FALSE, FALSE, FALSE,  FALSE, FALSE))
+  expect_equivalent(lapply(styles, check_style, nms = "iHave1Cat"    ), c(FALSE,  TRUE, FALSE, FALSE,  FALSE, FALSE))
+  expect_equivalent(lapply(styles, check_style, nms = "i_have_1_cat" ), c(FALSE, FALSE,  TRUE, FALSE,  FALSE, FALSE))
+  expect_equivalent(lapply(styles, check_style, nms = "i.have.1.cat" ), c(FALSE, FALSE, FALSE,  TRUE,  FALSE, FALSE))
+  expect_equivalent(lapply(styles, check_style, nms = "ihave1cat"    ), c(FALSE,  TRUE,  TRUE,  TRUE,   TRUE, FALSE))
+  expect_equivalent(lapply(styles, check_style, nms = "IHAVE1CAT"    ), c( TRUE, FALSE, FALSE, FALSE,  FALSE,  TRUE))
+  expect_equivalent(lapply(styles, check_style, nms = "I.HAVE_ONECAT"), c(FALSE, FALSE, FALSE, FALSE,  FALSE, FALSE))
 })
 
 test_that("linter ignores some objects", {
   # names for which style check is ignored
-  expect_lint("`%x%` <- t", NULL, object_name_linter(style="snake_case"))              # operator
-  expect_lint("`t.test` <- t", NULL, object_name_linter(style="ALLUPPERCASE"))         # std pkg
-  expect_lint(".Deprecated('x')", NULL, object_name_linter(style="alllowercase"))      # std pkg
-  expect_lint("abc::read_me <- t", NULL, object_name_linter(style="dotted.case"))      # ext pkg
-  expect_lint("as.foo <- t", NULL, object_name_linter(style="UpperCamelCase"))         # S3 generic
-  expect_lint("names.foo <- t", NULL, object_name_linter(style="UpperCamelCase"))      # int generic
-  expect_lint("sapply(x,f,USE.NAMES=T)", NULL, object_name_linter(style="snake_case")) # defined
-                                                                                       # elsewhere
+  expect_lint("`%x%` <- t", NULL, object_name_linter("snake_case"))              # operator
+  expect_lint("`t.test` <- t", NULL, object_name_linter("UPPERCASE"))         # std pkg
+  expect_lint(".Deprecated('x')", NULL, object_name_linter("lowercase"))      # std pkg
+  expect_lint("print.foo <- t", NULL, object_name_linter("CamelCase"))         # S3 generic
+  expect_lint("names.foo <- t", NULL, object_name_linter("CamelCase"))      # int generic
+  expect_lint("sapply(x,f,USE.NAMES=T)", NULL, object_name_linter("snake_case")) # defined elsewhere
 })
 
 test_that("linter returns correct linting", {
-  msg <- "Variable or function name should be lowerCamelCase."
-  linter <- object_name_linter(style="lowerCamelCase")
+  msg <- "Variable and function names should be in camelCase."
+  linter <- object_name_linter("camelCase")
```

Comment:
`camelCase` is used here, but there is a test "linter accepts vector of styles" that still uses `lowerCamelCase` (the message and the style for the latter test should be updated to refer to `camelCase`; and the `style` argument should probably be `styles`)

Reply:
The use of `lowerCamelCase` has been changed